MyOwnORM/MyOwnORM/Implementations/CustomDbSetRepository.cs

```diff
@@ -0,0 +1,523 @@
+﻿using MyOwnORM.Helper;
+using MyOwnORM.Interface;
+using MyOwnORM.Reflection;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM.Implementations
+{
+    public class CustomDbSetRepository<T> : ICustomDbSetRepository<T> where T : class
+    {
+        private string _connectionString;
+        private string tableName;
+        private string query;
+        private CustomDbSetService<T> dbSetExtension;
```

Comment:
apply 'readonly' to all services and helpers

Reply:
Fixed